react/week1/my-app/src/App.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import './App.css';
+
+function Header () {
+   return (
+    <header className="App-header">
+      <h1>Todo List</h1>
+    </header>
+   );
+}
+
+function List (props) {
+
+  //using idx key
+  const items = props.items.map((item, idx) => {
+    return <li key={idx}>{item.description}: {item.deadline}</li>
+  });
+
+  return (<ul>{items}</ul>);
+}
+
+function ListSection (props) {
```

Comment:
The ListItems array should be passed from the parent to this as a props. Right now it's passing state but I don't see anywhere where the state got the ListItems from?

Reply:
It kinda worked when I called it a state {state ? <ListSection items={state}/>
Not sure what should I do now ? :)

Reply:
I've implemented  changes so perhaps it will be possible for you to see them now.